exec/java-exec/src/main/java/org/apache/drill/exec/store/dfs/FormatSelection.java

```diff
@@ -63,6 +60,6 @@ public FileSelection getSelection(){
 
   @JsonIgnore
   public boolean supportDirPruning() {
```

Comment:
As above `support` --> `supports`. Is safe because this value is not serialized.

Reply:
Done.